app/assets/stylesheets/partials/_users.scss

```diff
@@ -0,0 +1,9 @@
+.profile_img{
```

Comment:
Selector `profile_img` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space